### ISSUE TITLE
0.49-DEV-StratCon - Campaign Status window doesn't refresh when reinforcing (Minor Issue)

### ISSUE THREAD

**Issue by Kailan7401** [MegaMek/mekhq#2514]
### Environment
StratCon alpha test build (MekHQ CI with Gradle #1168: Pull request #2144)
Windows 10
OpenJDK 11.0.10

### Description
Super minor thing, but when reinforcing a battle and using converting VP to SP, the numbers on the campaign status list on the right hand side of the map don't update until the page is tabbed out of/back into. This maybe probably also occurs when using SP without converting from VP, but I don't have any SP to test that at the moment.
![image](https://user-images.githubusercontent.com/32316785/114631700-13074c80-9cf8-11eb-93e5-446f68a5eb19.png)

### Files
Campaign save includes a pending base attack battle for ease of duplication
[Campaign Info.zip](https://github.com/MegaMek/mekhq/files/6304630/Campaign.Info.zip)
[megameklog.txt](https://github.com/MegaMek/mekhq/files/6304663/megameklog.txt)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/6304666/mekhqlog.txt)

**Comment by sixlettervariables** [comment]
Error from log:
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at mekhq.campaign.stratcon.StratconScenario.getInfo(StratconScenario.java:167)
	at mekhq.campaign.stratcon.StratconScenario.getInfo(StratconScenario.java:154)
	at mekhq.gui.stratcon.StratconScenarioWizard.setInstructions(StratconScenarioWizard.java:152)
	at mekhq.gui.stratcon.StratconScenarioWizard.setUI(StratconScenarioWizard.java:105)
	at mekhq.gui.stratcon.StratconScenarioWizard.setCurrentScenario(StratconScenarioWizard.java:90)
	at mekhq.gui.StratconPanel.actionPerformed(StratconPanel.java:650)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:369)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:1020)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:1064)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6635)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6400)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by NickAragua** [comment]
Fixed with commit  ba7898afc777a1f9f405a590c5abc91eeaa8242a
